content/automate/ManageIQ/Transformation/Common.class/__methods__/acquiretransformationhost.rb

```diff
@@ -31,6 +24,8 @@ def main
               @handle.log(:info, "Transformation Host: #{transformation_host.name}.")
               task.set_option(:transformation_host_id, transformation_host.id)
               task.set_option(:transformation_host_name, transformation_host.name)
+              task.set_option(:transformation_host_type, transformation_host_type)
```

Comment:
do you want to group all `transformation_host` related options under key `:transformation_host`?

Reply:
Probably a topic for `options` hash refactoring, isn't it ?